crypto/bn/bn_rsa_fips186_4.c

```diff
@@ -48,6 +48,38 @@ const BIGNUM ossl_bn_inv_sqrt_2 = {
     BN_FLG_STATIC_DATA
 };
 
+/*
+ * Refer to FIPS 186-5 Table B.2 for minimum rounds of Miller Rabin
+ * required for generation of RSA aux primes (p1, p2, q1 and q2)
+ * using an error probability of 2^–100.
+ */
```

Comment:
My preference would be to use the larger values - they are still much lower than the original 64 or more rounds. But I do not insist.

What does @kroeckx think?

Reply:
I'm not sure why we would want to use the values from table B.2 and limit the security strength to 2^-100. The values in table B.1 are calculated to match the security strength of generated key.

We have a copy of the function from C.1 documented at BN_prime_checks_for_size(), that return the same values as table B.1:
k:   140, security: 113 bits  (t: 38, M: 4)
k:   170, security: 129 bits  (t: 41, M: 4)
k:   200, security: 144 bits  (t: 44, M: 4)

Reply:
And for bn_rsa_fips186_5_prime_MR_rounds() it returns:
k:  1024, security: 116 bits  (t: 5, M: 25)
k:  1536, security: 130 bits  (t: 4, M: 35)
k:  2048, security: 146 bits  (t: 4, M: 39)